app/views/api/alerts/destroy.js.erb

```diff
@@ -1,3 +0,0 @@
-$(function(){
```

Comment:
I would actually keep this file (rewritten in JS if needed), and fix the Delete button by adding `remote: true`.
It used to be an AJAX action, but it was changed here: https://github.com/3scale/porta/pull/3636/files#diff-ab8780681158e67f54020924c99343f3c627228991ea13a9956221128a8615af
(not sure whether deliberately or not)